src/G2/Equiv/Tactics.hs

```diff
@@ -92,10 +120,14 @@ data PrevMatch t = PrevMatch {
   , container :: State t
 }
 
+-- TODO new constructor for lemma proving
+-- also have one for lemma application
 data ActMarker = Induction IndMarker
                | Coinduction CoMarker
                | Equality EqualMarker
                | NoObligations (StateET, StateET)
+               | LemmaProven (StateET, StateET)
```

Comment:
I added these two constructors while musing about having new ActMarkers for lemmas.  They're not used anywhere, so we might want to remove them for the time being.